tox.ini

```diff
@@ -6,12 +6,18 @@ isolated_build = true
 
 [testenv]
 extras = test
-deps = coverage
+deps =
+  coverage
+  mypy
 commands =
   coverage run -m pytest {posargs}
   coverage report
+  bash mypy_tests/run_local.sh mypy --strict check_negative_tests.py positive.py
+  bash mypy_tests/run_local.sh python check_negative_tests.py
```

Comment:
Done

Reply:
What I meant was that `check_negative_tests` should be integrated into the test suite. There's no reason it should be a separate script, and it still is even after your latest changes. It seems you did the opposite of what I wanted and removed the `mypy` call in favor of running the independent Python tests.

Reply:
Actually, running `mypy` against `positive.py` should be done in the same place where it's run against the negative tests. It would be consistent that way. At some point in the future I would probably want to run `mypy` against the main codebase too, but not yet.

Reply:
Ah I misinterpreted your original comment. I think I'm parsing it correctly now.  I like this version better.